cookbook/import_export/product-import.rst

```diff
@@ -0,0 +1,450 @@
+                type:  import
+                steps:
+                    validation:
+                        title: pim_connector.jobs.csv_product_import.validation.title
+                        class: %pim_connector.step.validator.class%
+                        services:
+                            charsetValidator: pim_connector.validator.item.charset_validator
+                    import:
+                        title:         pim_connector.jobs.csv_product_import.import.title
+                        services:
+                            reader:    pim_connector.reader.file.csv_product
+                            processor: pim_connector.processor.denormalization.product.flat
+                            writer:    pim_connector.writer.doctrine.product
+                    import_associations:
+                        title:         pim_connector.jobs.csv_product_import.import_associations.title
+                        services:
+                            reader:    pim_connector.reader.file.csv_association
+                            processor: pim_connector.processor.denormalization.product_association.flat
+                            writer:    pim_connector.writer.doctrine.product_association
+
+With the ``type`` parameter, we can see that this job is an import.
+
+We can also count 3 defined steps, ``validation``, ``import`` and ``import_associations``.
+
+Charset Validation Step
+-----------------------
+
+The purpose of this step it to validate that the input file has the expected encoding (UTF-8 by default).
+
+This step is a custom step, not a default ``ItemStep``, so we need to define the custom class to use with the parameter ``class`` and the value ``%pim_connector.step.validator.class%``.
+
+.. code-block:: yaml
+
+    [...]
+    validation:
+        title: pim_connector.jobs.csv_product_import.validation.title
+        class: %pim_connector.step.validator.class%
+        services:
+            charsetValidator: pim_connector.validator.item.charset_validator
+    [...]
+
+This parameter ``pim_connector.step.validator.class`` is defined in ``src\Pim\Bundle\ConnectorBundle\Resources\config\steps.yml``.
+
+.. code-block:: yaml
+
+    parameters:
+        pim_connector.step.validator.class: Pim\Component\Connector\Step\ValidatorStep
+
+We can also see that we inject a service ``pim_connector.validator.item.charset_validator`` in this step.
+
+This service is defined in ``src\Pim\Bundle\ConnectorBundle\Resources\config\items.yml``.
+
+.. code-block:: yaml
+
+    parameters:
+        pim_connector.validator.item.charset_validator.class: Pim\Component\Connector\Item\CharsetValidator
+
+    services:
+        pim_connector.validator.item.charset_validator:
+            class: %pim_connector.validator.item.charset_validator.class%
+
+
+The constructor of the ``CharsetValidator`` show that it's configured to check only a file which match some extensions and to check the 'UTF-8' encoding.
```

Comment:
Missing some verbs conjugations: "The constructor of the ``CharsetValidator`` shows that it's configured to check only a file which matches some extensions and to check the 'UTF-8' encoding."